README.md

```diff
@@ -26,9 +26,11 @@ steps:
       restore_cache $(hash_file package-lock.json)
 
     plugins:
-      - automattic/a8c-ci-toolkit#v2.14.0
+      - automattic/a8c-ci-toolkit#2.15.0
 ```
 
+Don't forget to verify what [the latest release](https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/releases/tag/2.15.0) is and use that value instead of `2.15.0`.
```

Comment:
I think linking the "latest release" to the `2.15.0` would be counter-productive. Should we use the link for the releases page instead?

```suggestion
Don't forget to verify what [the latest release](https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/releases/) is and use that value instead of `2.15.0`.
```

Reply:
+1

Reply:
Good point. I'll one-up you on that and use https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/releases/latest

I discovered the existence of `/latest` when working on this change. But as you can see by following the link, it redirects to the actual latest URL. So I copy pasted that URL believing it was `/latest` _and didn't notice it..._ 😅

Reply:
```suggestion
Don't forget to verify what [the latest release](https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/releases/latest) is and use that value instead of `2.15.0`.
```